src/backend/gporca/libgpopt/src/translate/CTranslatorDXLToExpr.cpp

```diff
@@ -574,8 +574,13 @@ CTranslatorDXLToExpr::PexprLogicalGet(const CDXLNode *dxlnode)
 
 	CTableDescriptor *ptabdesc = Ptabdesc(table_descr);
 
-	CWStringConst strAlias(m_mp,
-						   table_descr->MdName()->GetMDName()->GetBuffer());
+	const gpmd::CMDName *ptabalias = table_descr->MdName();
+	if (NULL != table_descr->MdAlias())
```

Comment:
Debug print is human-readable and  is not used in tests usually, besides the case I made specially for it. Reason for the GUC was successful passage of unrelated tests, and since tests are passed, there is no reason to use GUC here.

Reply:
I think there will be a discrepancy between expected and actual behavior.

Reply:
Of course it will. It's just not used in the tests.